### PR TITLE
initial support for RPMs

### PR DESCRIPTION
**Pull request by bobcallaway**
Signed-off-by: Bob Callaway <bcallawa@redhat.com>

### CODE REVIEW

File: tests/pgp.go [hunk 1]
```diff
@@ -150,13 +149,12 @@ func init() {
...
 }
 
-func Sign(t *testing.T, m io.Reader) []byte {
-	t.Helper()
-	var b bytes.Buffer
-	if err := openpgp.ArmoredDetachSign(&b, keys[0], m, nil); err != nil {
-		t.Fatal(err)
+func Sign(b []byte) ([]byte, error) {
+	var buf bytes.Buffer
+	if err := openpgp.DetachSign(&buf, keys[0], bytes.NewReader(b), nil); err != nil {
```
Comment (dlorenc) [inline tests/pgp.go:154]: Any reason for this change?

File: cmd/cli/app/root.go [hunk 2]
```diff
@@ -82,8 +82,9 @@ func initConfig() {
 	viper.SetEnvPrefix("rekor")
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Println("Using config file:", viper.ConfigFileUsed())
+	if err := viper.ReadInConfig(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```
Comment (dlorenc) [inline cmd/cli/app/root.go]: It looks like this one might be breaking the tests.

Comment (bobcallaway) [reply cmd/cli/app/root.go]: yeah, you're right.

File: tests/pgp.go (hunk 1 above)
Comment (bobcallaway) [reply tests/pgp.go:154]: RPMs can only use Binary signatures, not armored. I don't think it mattered for the tests already written.

Comment (dlorenc) [reply tests/pgp.go:154]: Ah thanks.

**Review by dlorenc — APPROVED** [review]

**Review by dlorenc — APPROVED** [review]

**Pull request merged**